src/pymovements/gaze/io.py

```diff
@@ -360,6 +362,98 @@ def from_asc(
             for fileinfo_key, fileinfo_dtype in column_schema_overrides.items()
         ])
 
+    if experiment is None:
+        experiment = Experiment(sampling_rate=metadata['sampling_rate'])
+    if experiment.screen is None:
+        experiment.screen = Screen()
+    if experiment.eyetracker is None:
+        experiment.eyetracker = EyeTracker()
+
+    # Resolution comparison with ASCII file and experiment
+
+    if experiment.screen.width_px is None or experiment.screen.height_px is None:
```

Comment:
leave a comment that you assume that width and height are always missing mutually.